jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/AwsExplorerServiceNode.kt

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer.nodes
+
+import com.intellij.openapi.project.Project
+
+interface AwsExplorerServiceNode {
```

Comment:
Do we need both `AwsExplorerServiceNode` and `AwsExplorerServiceRootNode` - can we collapse them together?